content/applications/inventory_and_mrp/manufacturing/management/bill_configuration.rst

```diff
@@ -1,94 +1,121 @@
-=========================
-Create Bills of Materials
-=========================
+================================
+Create a Bill of Materials (BoM)
+================================
 
-A *Bill of Materials* is a document defining the quantity of each
-component required to manufacture a finished product. It also includes
-the routing and individual steps of the manufacturing process.
+A *Bill of Materials* (or *BoM*) is a document that defines the quantity of each component required
+to manufacture or deliver a finished product. It can also include various operations types and
+steps, and instructions for the individual step guidelines needed to complete a production process.
 
-With Odoo, you can link multiple BoMs to each product and use it to
-describe multiple variants of them. Each BoM will, yet, be associated
-with one product only.
+In Odoo Manufacturing, multiple :abbr:`BoMs (Bills of Materials)` can be linked to a product, so
+that even product variants can have their own specific :abbr:`BoMs (Bills of Materials)`.
 
-This feature will help optimize your manufacturing process while saving
-you time.
+Correctly setting up a Bill of Materials helps optimize the manufacturing process, and saves time.
```

Comment:
Correctly setting up a :abbr:`BoM (Bill of Materials)` helps optimize the manufacturing process, and saves time.

Would this use of ":abbr:" work in this instance? Not quite sure - totally your call.

Reply:
Yes, it would work in this instance and help keep things consistent from the start. Good catch!